core/pkg/resultshandling/printer/v2/utils.go

```diff
@@ -82,3 +86,12 @@ func finalizeResources(results []resourcesresults.Result, allResources map[strin
 	}
 	return resources
 }
+
+func CheckShortTerminalWidth() bool {
+	width, _, err := term.GetSize(int(os.Stdin.Fd()))
+	if err != nil {
+		// Default to larger output table
+		return false
+	}
+	return width <= 120
+}
```

Comment:
I had hoped we could calculate the width the table is going to be, and then use that as input to this function: if the table can be displayed within the existing width, do so, else go to short mode.

Reply:
A problem that I had felt with that approach was the fact that we'd have to figure out the width of the largest row to determine this, which might lead to some redundancy, but on second thought, I guess there might be a way if we slightly alter the order in which we construct the table, I'll take a shot with the approach and update the results here.